debug/projections.html

```diff
@@ -90,20 +90,20 @@
         albers: 3,
         equalEarth: 1.0,
         equirectangular: 1.0,
-        lambert: 1.0,
+        lambertConicConformal: 1.0,
```

Comment:
Perhaps swap to `lambertConformalConic`? It seems like this is a more common order in the name (i.e. Wikipedia, GIS products etc.).